src/iiifRegion.js

```diff
@@ -8,10 +8,6 @@ var IiifRegion = function(options, dispatcher) {
   this.y = 0 || options.y;
   this.width = 0 || options.width;
   this.height = 0 || options.height;
-  this.rotation = options.rotation || 0;
-  this.scale = options.scale || 'full';
-  this.quality = options.quality || 'default';
-  this.format = options.format || 'jpg';
```

Comment:
I can see a value in leaving these here as the defaults, rather than hiding the default properties of the iiif region in a getter function. The idea with the iiif region object was to represent the state of the iiif selection and allow it to be changed/updated. The event emitter can then emit events whenever a setter terminates, so multiple subscribers can react to this state change.

Having information about the region shared in 3 different objects (iiifRegion, selection, and transformSelection) is a good way to get them out of synch, and makes it difficult to know what the "authoritative" record is.

I'm going to merge these two PRs because working code is a good thing, but I will try to do a minor refactor tonight to show a different way to implement it.